packages/crawler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "crawler",
+  "version": "0.0.0",
+  "private": true,
+  "license": "UNLICENSED",
+  "scope": "@porsche-design-system",
+  "author": "Dr. Ing. h.c. F. Porsche AG",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "test:e2e:puppeteer": "jest --colors -c tests/e2e/config/jest.config.ts",
+    "test:e2e": "concurrently -k -s 'first' -n 'cdn,puppeteer' 'serve-cdn' 'yarn test:e2e:puppeteer'"
+  },
+  "dependencies": {
+    "@porsche-design-system/shared": "0.0.0",
+    "@porsche-design-system/components-js": "0.0.0",
+    "puppeteer": "~19.3.0",
+    "ts-node": "^10.9.1",
+    "typescript": "~4.9.3"
+  },
+  "devDependencies": {}
```

Comment:
`@porsche-design-system/assets` package is missing and necessary to provide the `serve-cdn` executable.

Reply:
Added this package